src/java/fr/paris/lutece/plugins/notifygru/modules/forms/services/provider/FormsProvider.java

```diff
@@ -288,6 +295,21 @@ public Collection<InfoMarker> provideMarkerValues( )
         InfoMarker updateStatusDateMarker = new InfoMarker( MARK_STATUS_UPDATE_DATE );
         updateStatusDateMarker.setValue( formResponse.getUpdateStatus( ).toString( ) );
         result.add( updateStatusDateMarker );
+
+        SimpleDateFormat timeFormat = new SimpleDateFormat("HH:mm:ss");
+        String creationtime =  timeFormat.format(formResponse.getCreation( ));
+        InfoMarker creationTimeMarker = new InfoMarker( MARK_CREATION_TIME );
+        creationTimeMarker.setValue(creationtime);
+        result.add( creationTimeMarker );
+
+        String updatetime =  timeFormat.format(formResponse.getUpdate());
+        InfoMarker updateTimeMarker = new InfoMarker( MARK_UPDATE_TIME );
+        updateTimeMarker.setValue(updatetime);
```

Comment:
Please send a "time" object to the template, the format of the date will be chosen in the template